### DISCUSSION TITLE
[1.4.0] Double clique sur un widget

### DISCUSSION THREAD

**Discussion by syrm** [skarab42/marv#142] [Ideas]
Quand on est en mode édition le double clique sur un widget devrait éditer celui-ci, actuellement ça rajoute un widget

**Comment by Arubinu** [comment]
J'ai essayé de me pencher sur le problème, mais j'ai encore du mal avec Marv.
Le seul moyen que j'avais trouvé était de modifier l'objet `widget` pour y ajouter l'accès à la méthode `editAction`, mais je trouve ça extrêmement dégueulasse ^^

Je laisse du coup la main à quelqu'un qui a plus de facilité là dedans :/
